notebooks/All About REM Statistics.ipynb

```diff
@@ -426,16 +483,15 @@
    "id": "d43d13ea-a01f-4b6f-bac6-2e1c2bc242aa",
    "metadata": {},
    "source": [
-    "Unlike the mean, the median of the distribution uses no fuel oil after the upgrade. This is because whatever\n",
-    "home or homes in the sample used fuel oil for something other than heating, there were probably not very\n",
-    "many of them.\n",
+    "Unlike the mean, the median of the distribution uses no fuel oil after the upgrade. This is because a \n",
+    "minority of homes in the sample used fuel oil for things other than heating (in this case, we know it was 45%). \n",
     "\n",
     "Indeed, we can look at the 20th percentile and see that it is also still zero after the upgrade."
```

Comment:
@vengroff I think maybe you meant to say the 80th here? Because if the median is 0, then the 20th is obviously also 0. But in this case, the 80th is not 0 because as I mentioned, 45% of samples have fuel oil water heating.

Reply:
Fixed. I now mention both.